test/ANNOTATIONS.yaml

```diff
@@ -1088,6 +1088,8 @@ LCALS-raw-short: &lcals-base
     - Task intents to operate on fields of 'this' (#13840)
   09/10/19:
     - Restrict applicability of field intents (#14057)
+  02/14/23:
+    - LCALS perf back to normal (Cray/chapel-private#4228)
```

Comment:
Speaking of this, I noticed over the weekend that the performance has popped back up to the other level again... :(